nixos/modules/services/hardware/amdgpu.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.hardware.amdgpu;
+in {
+  options.hardware.amdgpu = {
+    legacySupport.enable = lib.mkEnableOption ''
+      support for old AMD graphics cards, this uses `amdgpu`
```

Comment:
Maybe "using the `amdgpu` driver instead of `radeonsi` for [this specific hardware]", etc?

Reply:
Also, it should really say "kernel module", not "driver".

Reply:
Please note the difference between `radeon` and `radeonsi`. One is a kernel driver for old Radeon cards and the other a modern OpenGL driver for all sorts of Radeon cards, including ones powered by AMDGPU.

Reply:
Then I suppose it should say "`radeon` kernel driver" instead here.